### ISSUE TITLE
[CalendarReminder.json>Snooze for]: Voice access user is not able to access the 'Snooze for' combo box through move to 'Snooze for' command under 'CalendarReminder.json'.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7518]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25126.1000)
Application: Adaptive Cards UWP
Application Version: 3.0.2206.2001
Tool: Voice access

**Repro Steps:**
1. Turn on Voice access.
2. Open Adaptive Cards UWP application.
3. Command: Show Numbers
4. Select particular number for 'CalendarReminder.json' 
5. Command: Move to Snooze for
6. Observe the issue.

**Actual Result:**
Voice access user is not able to access the 'Snooze for' combo box through move to 'Snooze for' command under 'CalendarReminder.json'.

**Expected Result:**
Voice access user should be able to access the 'Snooze for' combo box through move to 'Snooze for' command under 'CalendarReminder.json'.

**Observation:**
'Click Snooze for' command is also not working.

**User Impact:**
People with limited mobility (cerebral palsy, quadriplegia) who use Voice Access will get impacted as they will not be able to interact with the control easily and accessing through numbers would take more time for them.

**MAS reference link:**
[[No Disruption of Accessibility Features - Liquid (microsoft.com)]](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.02.01#Zguide)

**Have feedback to share on Bugs?** Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

<img width="943" alt="MAS4 3 1 'Move to Snooze for ' command is not getting performed" src="https://user-images.githubusercontent.com/104483005/173423505-dc238f45-0ea1-4f15-87a5-e48e16f1cb7f.png">

https://user-images.githubusercontent.com/104483005/173425823-76f4029e-5172-4974-a9b1-db818cc1448a.mp4

### Card JSON

```JSON
NA
```

**Comment by paulcam206** [comment]
To whoever starts work on this issue: we should remove all of the samples that should be using labels (clicking "Snooze for" would work here if it the sample correctly supplied it as a label, as we do with `CalendarReminderWithLabels.json`).

**Comment by vaishali1397** [comment]
Issue has been fixed in below test environment, hence adding A11yTTValidated tag.
**Test Environment:**
OS Version: 22H2 (Build 25330.1000)
Application: Adaptive Card Visualizer

**
https://user-images.githubusercontent.com/104483005/235418224-4f3927ba-3216-4dff-b798-1908f8d323ee.mp4
**: Adaptive Cards~ Issue fixed snooze for combobox.mp4
